packages/driver/src/cy/overrides.ts

```diff
@@ -1,9 +1,13 @@
 import _ from 'lodash'
-// @ts-ignore
 import { registerFetch } from 'unfetch'
```

Comment:
`//@ts-ignore` is unnecessary because the type of `registerFetch` is added with `patch-package`.